nx-X11/programs/Xserver/hw/nxagent/Error.c

```diff
@@ -323,64 +323,53 @@ void nxagentEndRedirectToClientsLog(void)
   nxagentCloseClientsLogFile();
 }
 
+/*
+ * returns a pointer to the static nxagentHomeDir. The caller must not free
+ * this pointer!
+ */
```

Comment:
Where are you freeing nxagentHomeDir, finally, then? Leaving it to the application tidy up at end of application run?

Reply:
well, nxagentHoemdir is supposed to last during the nxagent runtime. Cleaning up such global variables does not have a place right now. Maybe ddxGiveUp() or AbortDDX(), not sure.